### ISSUE TITLE
logging.basicConfig should not be called by 'import bokeh'

### ISSUE THREAD

**Issue by rizar** [bokeh/bokeh#1973]
The `logging.basicConfig` call is typically performed by the user. It is allowed to be called only once during program life time, all subsequent calls do nothing.

For these reason I think it is not nice to have it done implicitly by `import bokeh`. It breaks any application using `bokeh` that configures logging after loading used modules.

The respective line in the code: https://github.com/bokeh/bokeh/blob/master/bokeh/__init__.py#L12)

**Comment by bryevdv** [comment]
Hrmm not sure when that got put in there. It probably ought to be off by default, but configurable with an environment variable. It is often desirable to run the examples with logging on, but would be a tedious burden to have to edit the file every time beforehand.

**Comment by bryevdv** [comment]
suggest `BOKEH_PYTHON_LOGGING` handled in `settings.py`

**Comment by damianavila** [comment]
>suggest BOKEH_PYTHON_LOGGING (yes/no) handled in settings.py

I agree with this solution...

**Comment by bryevdv** [comment]
As an aside, we nee to make the env var names more consistent in terms of what configures the python side and what configures the JS side. e.g, `BOKEH_LOG_LEVEL`  should probably be `BOKEHJS_LOG_LEVEL`

**Comment by damianavila** [comment]
>As an aside, we nee to make the env var names more consistent

:+1:
